src/notation/view/widgets/editstaff.cpp

```diff
@@ -213,7 +213,12 @@ void EditStaff::updateInstrument()
 
     longName->setPlainText(m_instrument.nameAsPlainText());
     shortName->setPlainText(m_instrument.abbreviatureAsPlainText());
-    instrumentName->setText(m_instrument.nameAsPlainText());
+    const InstrumentTemplate* templ = mu::engraving::searchTemplate(m_instrument.id());
+    if (templ) {
+        instrumentName->setText(formatInstrumentTitle(templ->trackName, templ->trait));
+    } else {
+        instrumentName->setText(qtrc("EditStaffBase", "Unknown"));
```

Comment:
Judging by nearby files, I think the `qtrc` part of this line should be `qtrc("notation/editstaff", "Unknown")`.

The first argument to `qtrc` is a context string used for two purposes:

1. Give translators a clue about where the string appears in the program.
2. Enable the second argument (i.e. the main string) to be translated differently in different areas of the program.

Class names are often used for the context string but we are using source file paths for some reason.

Reply:
Done and done!